scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -139,189 +139,7922 @@
+	language	Estonian Sign Language	eso	1878	eso
+	language	Esuma	esm	1879	esm
+	language	Etchemin	etc	1880	etc
+	language	Etebi	etb	1881	etb
+	language	Eten	etx	1882	etx
+	language	Eteocretan	ecr	1883	ecr
+	language	Eteocypriot	ecy	1884	ecy
+	language	Ethiopian Sign Language	eth	1885	eth
+	language	Etkywan	ich	1886	ich
+	language	Eton (Cameroon)	eto	1887	eto
+	language	Eton (Vanuatu)	etn	1888	etn
+	language	Etruscan	ett	1889	ett
+	language	Etulo	utr	1890	utr
+	language	Eudeve	eud	1891	eud
+	language	Evant	bzz	1892	bzz
+	language	Even	eve	1893	eve
+	language	Evenki	evn	1894	evn
+	language	Eviya	gev	1895	gev
+	language	Ewage-Notu	nou	1896	nou
+	language	Ewe	ewe	1897	ewe	ee
+	language	Ewondo	ewo	1898	ewo
+	language	Extremaduran	ext	1899	ext
+	language	Eyak	eya	1900	eya
+	language	Ezaa	eza	1901	eza
+	language	Fa d'Ambu	fab	1902	fab
+	language	Fagani	faf	1903	faf
+	language	Faifi	fif	1904	fif
+	language	Faire Atta	azt	1905	azt
+	language	Faita	faj	1906	faj
+	language	Faiwol	fai	1907	fai
+	language	Fala	fax	1908	fax
+	language	Falam Chin	cfm	1909	cfm
+	language	Fali	fli	1910	fli
+	language	Faliscan	xfa	1911	xfa
+	language	Fam	fam	1912	fam
+	language	Fanagalo	fng	1913	fng
+	language	Fanamaket	bjp	1914	bjp
+	language	Fanbak	fnb	1915	fnb
+	language	Fang (Cameroon)	fak	1916	fak
+	language	Fang (Equatorial Guinea)	fan	1917	fan
+	language	Fania	fni	1918	fni
+	language	Fanti	fat	1919	fat
+	language	Far Western Muria	fmu	1920	fmu
+	language	Farefare	gur	1921	gur
+	language	Faroese	fao	1922	fao	fo
+	language	Fas	fqs	1923	fqs
+	language	Fasu	faa	1924	faa
+	language	Fataleka	far	1925	far
+	language	Fataluku	ddg	1926	ddg
+	language	Fayu	fau	1927	fau
+	language	Fe'fe'	fmp	1928	fmp
+	language	Fembe	agl	1929	agl
+	language	Fernando Po Creole English	fpe	1930	fpe
+	language	Feroge	fer	1931	fer
+	language	Fiji Hindi	hif	1932	hif
+	language	Fijian	fij	1933	fij	fj
+	language	Filipino	fil	1934	fil
+	language	Filomena Mata-Coahuitlán Totonac	tlp	1935	tlp
+	language	Finland-Swedish Sign Language	fss	1936	fss
+	language	Finnish	fin	1937	fin	fi
+	language	Finnish Sign Language	fse	1938	fse
+	language	Finongan	fag	1939	fag
+	language	Fipa	fip	1940	fip
+	language	Firan	fir	1941	fir
+	language	Fiwaga	fiw	1942	fiw
+	language	Flaaitaal	fly	1943	fly
+	language	Flinders Island	fln	1944	fln
+	language	Foau	flh	1945	flh
+	language	Foi	foi	1946	foi
+	language	Foia Foia	ffi	1947	ffi
+	language	Folopa	ppo	1948	ppo
+	language	Foma	fom	1949	fom
+	language	Fon	fon	1950	fon
+	language	Fongoro	fgr	1951	fgr
+	language	Foodo	fod	1952	fod
+	language	Forak	frq	1953	frq
+	language	Fordata	frd	1954	frd
+	language	Fore	for	1955	for
+	language	Forest Enets	enf	1956	enf
+	language	Fortsenal	frt	1957	frt
+	language	Francisco León Zoque	zos	1958	zos
+	language	Frankish	frk	1959	frk
+	language	French	fra	1960	fra	fre	fr
+	language	French Sign Language	fsl	1961	fsl
+	language	Friulian	fur	1962	fur
+	language	Fula, Fulah, Pulaar, Pular, Fulah, Pulaar, Pular	ful	1963	ful	ff	Fula	Fulah	Pular	Pulaar
```

Comment:
Why does `ful` have seven variations? At https://en.wikipedia.org/wiki/ISO_639:f they give it a single English name.

Perhaps the release note could indicate the source of the data.

I downloaded a zip from https://iso639-3.sil.org/code_tables/download_tables#Complete%20Code%20Tables (I'm not sure if this is the right place) and I only see one name, not seven:

```
pdurbin@beamish ~ % cd ~/Downloads/iso-639-3_Code_Tables_20240415 
pdurbin@beamish iso-639-3_Code_Tables_20240415 % ls
iso-639-3-macrolanguages.tab	iso-639-3.tab			iso-639-3_Name_Index.tab	iso-639-3_Retirements.tab
pdurbin@beamish iso-639-3_Code_Tables_20240415 % ack Fulah
iso-639-3_Name_Index.tab
2069:ful	Fulah	Fulah

iso-639-3.tab
1978:ful	ful	ful	ff	M	L	Fulah
```

(Later I noticed that "Divehi, Dhivehi, Maldivian, Dhivehi" has four values.)

Is there a script we can add to this pull request that was used to put these values into the tsv file? What process do we follow if we need to update the list in a few years? Should we put something in the dev guide?

Reply:
Already fixed it. Or at least most.

Reply:
We had some of these aliases like Pular in our old block - possibly from their 639-1 /639-2 sources. It looks like the merge method resulted in some duplicates (like Pular showing twice) which are what looks like it got fixed. There's a further issue though with this example and perhaps others: Pular is listed separately as
fuf				I	L	Pular	
in the ISO file and Pulaar is
fuc				I	L	Pulaar	

Right now, I don't see fuf or fuc in the citation block and if they were there (I think they should be), they shouldn't still be listed as alternates for ful.

I'm not sure how to automate unscrambling that. Almost seems like manually going through the existing 187 entries to see if they've been split or dropped, etc. is going to be needed. (Maybe automation can pull out all the lines where we have extra alternates so those can be the ones to focus on.)

Reply:
I merge with our original tsv file. That file has Pulaar mapped to ful

language	Fula, Fulah, Pulaar, Pular	ful	48	ful	ff	Fula	Fulah	Pulaar	Pular

Reply:
I added the source of the ISO file in the release notes